### ISSUE TITLE
dotfiles/bin/dcu breaks docker-compose commands

### ISSUE THREAD

**Issue by kurie** [gfredericks/dotfiles#1]
specifically `docker-compose ps` and `docker-compose logs`.  Maybe others.

**Comment by gfredericks** [comment]
yeah this seems tricky, because part of the idea is that `dcu` starts an isolated set of components, so it's not clear how you would figure out what components are being talked about in a separate call to `docker-compose` or something similar.
